src/components/dashboard/components/SideNavigation.jsx

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
-import { Link } from 'react-bootstrap';
 import AddDictionary from './dictionary/AddDictionary';
-
+import{ Link }from 'react-router-dom'
```

Comment:
eslint spaces on the import

Reply:
I will deal with that thank you for the feedback